robolectric-shadows/shadows-core/src/main/java/org/robolectric/fakes/RoboMenu.java

```diff
@@ -52,7 +51,7 @@ public MenuItem add(int groupId, int itemId, int order, CharSequence title) {
 
   @Override
   public MenuItem add(int groupId, int itemId, int order, int titleRes) {
-    return add(groupId, itemId, order, context.getResources().getString(titleRes));
+    return add(groupId, itemId, order, RuntimeEnvironment.application.getResources().getString(titleRes));
```

Comment:
Two reasons:
1. I wasn't able to get that to work appropriately in all the test cases I was using.
2. This conforms to how RoboSubMenuItem already works and is now more consistent. I feared that change would create more inconsistency, whereas this code never worked properly and resulted in an NPE.